adafruit_connection_manager.py

```diff
@@ -58,6 +58,10 @@ def __init__(self, socket: CircuitPythonSocketType, tls_mode: int) -> None:
         self.recv = socket.recv
         self.close = socket.close
         self.recv_into = socket.recv_into
+        if hasattr(socket, "_interface"):
+            self._interface = socket._interface
+        if hasattr(socket, "_socket_pool"):
+            self._socket_pool = socket._socket_pool
```

Comment:
When do these attributes get used? I could not find where they are used.

Reply:
They are used [here](https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/blob/06281a57ee77fdc6416587cf3656b0aff586a72d/adafruit_esp32spi/adafruit_esp32spi_socketpool.py#L90), when using the fake socket. The `_socket_pool` and `_interface` need to be copied over so the duplicate socket works (before everything was global)

Reply:
Got it. Could you add a comment about that?

Reply:
fixed here: b14ed99